gotagger.go

```diff
@@ -65,6 +66,12 @@ type Config struct {
 	// if there are no new commits, but the worktree is "dirty".
 	DirtyWorktreeIncrement string
 
+	// IncrementConfigFile path to the file where the increment configs are defined.
+	IncrementConfigFile string
```

Comment:
Let's just go ahead and bite the config file bullet: add json tags to the Config struct and Unmarshal it from the config file. I'll take a stab at that...